sklbench/benchmarks/sklearn_estimator.py

```diff
@@ -414,12 +424,28 @@ def measure_sklearn_estimator(
                         data_args = (x_train,)
                     else:
                         data_args = (x_test,)
-                batch_size = get_bench_case_value(
-                    bench_case, f"algorithm:batch_size:{stage}"
-                )
-                if batch_size is not None:
+
+                if method == "partial_fit":
+                    num_batches = get_bench_case_value(bench_case, "data:num_batches")
+                    batch_size = get_bench_case_value(bench_case, "data:batch_size")
+
+                    if batch_size is None:
+                        if num_batches is None:
+                            num_batches = 5
+                        batch_size = (
+                            data_args[0].shape[0] + num_batches - 1
+                        ) // num_batches
+                    if num_batches is None:
+                        num_batches = (
+                            data_args[0].shape[0] + batch_size - 1
+                        ) // batch_size

```

Comment:
Batching is regulated by `batch_size` only.
All default parameters should be defined in `get_bench_case_value`.

Reply:
> Batching is regulated by `batch_size` only.
why do you think this is strict rule?

Reply:
> All default parameters should be defined in `get_bench_case_value`.

here num_batches is set to default only if both num_batches and batch_size are None. such logic can't be implemented via get_bench_case_value as far as I understand